src/cdk/app.ts

```diff
@@ -33,29 +59,29 @@ export class MiraApp {
    * @param {String} fileName (Optional) Can provide an arbitary name to
    * lookup if name exists in configs.
    */
-  async getStack (fileName: string): Promise<Stack | boolean> {
+  async getStack (fileName: string): Promise<MiraValidStack | boolean> {
```

Comment:
Again wondering if there will be overlap if we remove `class MiraStack extends cdk.Stack` and compose instead.